uiowa/uiowa
# [Facilities] Alert updates

Relates to https://github.com/uiowa/uiowa/issues/6880. 

Templates:
- Branded Template: `UI 1 row x 1 col + 1 row x 2 cols` https://apps.its.uiowa.edu/dispatch/communications/1238605612
- Template with conditionals that don't seem to be working: https://apps.its.uiowa.edu/dispatch/templates/1238199082
  - Test out at https://facilities.prod.drupal.uiowa.edu/node/3501/dispatch and adding your email address to https://apps.its.uiowa.edu/dispatch/communications/1238605612



# How to test


## Test on facilities

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli 
```

- Confirm that the title is not displaying on the alert node edit form only in Facilities split
- Confirm that the category field is required only in Facilities split
- Confirm https://apps.its.uiowa.edu/dispatch/templates/1238199082 is using conditionals for all fields and that project number appears at the bottom of the email. 
- Confirm that project number has been added to the bottom of the branded email template https://apps.its.uiowa.edu/dispatch/communications/1238605612. 

## Test on default

```
ddev blt ds --site=default && ddev drush @default.local config-split:activate sitenow_alerts -y && ddev drush @default.local uli node/add/alert
```

- Confirm that title is still displaying on node edit form for alert
- Confirm that category field is not required
